flood_adapt/dbs_controller.py

```diff
@@ -927,14 +927,28 @@ def save_projection(self, projection: IProjection) -> None:
             raise ValueError(
                 f"'{projection.attrs.name}' name is already used by another projection. Choose a different name"
             )
-        else:
-            (self.input_path / "projections" / projection.attrs.name).mkdir()
-            projection.save(
-                self.input_path
-                / "projections"
-                / projection.attrs.name
-                / f"{projection.attrs.name}.toml"
+
+        projection_path = Path(self.input_path / "projections" / projection.attrs.name)
+        os.mkdir(projection_path)
+
+        # Handle user uploaded shapefile
+        if projection.attrs.socio_economic_change.new_development_shapefile is not None:
+            src_file = Path(
+                projection.attrs.socio_economic_change.new_development_shapefile
             )
+            src_file_name = src_file.name
+
+            dst_path = projection_path / src_file_name
+            # OLD OPTION: dst_path = projection_path / "new_development_area.geojson"
+            # This is what it was, do we want to have all user provided shapefiles named the same?
+            # There will be only one per projection so its okay?
```

Comment:
Maybe check this with @GundulaW. If it is indeed only possible to upload 1 file, the best option would be {projection.attrs.name}.geojson I think. Most consistent with the other stuff we save

Reply:
Indeed there is only 1 file that can have multiple polygons